packages/core/src/framebuffer/FramebufferSystem.js

```diff
@@ -427,6 +426,36 @@ export default class FramebufferSystem extends System
         }
     }
 
+    forceStencil()
```

Comment:
You could add some documentation, even if private? Please explain why and how it's used (not what it is)

Reply:
yep